dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -209,9 +214,9 @@ public void testComponent_failure() throws IOException, ValidityException, Parsi
       Nodes presDependencyMediation = document.query("//pre[@class='suggested-dependency-mediation']");
       Assert.assertTrue("For failed component, suggested dependency should be shown",
           presDependencyMediation.size() >= 1);
-      Nodes presDependencyTree = document.query("//p[@class='DEPENDENCY_TREE_NODE']");
+      Nodes dependencyTree = document.query("//p[@class='DEPENDENCY_TREE_NODE']");
       Assert.assertTrue("Dependency Tree should be shown in dashboard even when FAILED",
-          presDependencyTree.size() > 0);
+          dependencyTree.size() > 0);
```

Comment:
`assertWithMessage("Dependency Tree...").that(dependencyTree).isNotEmpty();`

I hope `Nodes` implements `Iterable<Node>`.